inst/rmarkdown/templates/oup/resources/template.tex

```diff
@@ -1,8 +1,31 @@
+% Colors for colorlinks
 $if(colorlinks)$
 \PassOptionsToPackage{dvipsnames,svgnames*,x11names*}{xcolor}
 $endif$
 
-\documentclass[$if(fontsize)$$fontsize$,$else$12pt,$endif$$if(spacing)$$spacing$,$else$halfline,$endif$$if(papersize)$$papersize$,$else$a4paper,$endif$$for(classoptions)$$classoptions$$sep$,$endfor$]{ouparticle}
+% Set document class options
+\documentclass[webpdf$if(papersize)$,$papersize$$else$,large$endif$$if(document_style)$,$document_style$$else$,contemporary$endif$$if(unnumsec)$,unnumsec$endif$$if(namedate)$,namedate$endif$$if(classoptions)$,$for(classoptions)$$classoptions$$sep$,$endfor$$endif$]{oup-authoring-template}
+
+% one column
+$if(onecolumn)$\onecolumn$endif$
+
+%\usepackage{showframe}
+
+% line numbers
+$if(numberlines)$\usepackage{lineno}\linenumbers$endif$
+
+% Theorem stuff from template
+\theoremstyle{thmstyleone}%
+\newtheorem{theorem}{Theorem}%  meant for continuous numbers
+%%\newtheorem{theorem}{Theorem}[section]% meant for sectionwise numbers
+%% optional argument [theorem] produces theorem numbering sequence instead of independent numbers for Proposition
+\newtheorem{proposition}[theorem]{Proposition}%
+%%\newtheorem{proposition}{Proposition}% to get separate numbers for theorem and proposition etc.
+\theoremstyle{thmstyletwo}%
+\newtheorem{example}{Example}%
+\newtheorem{remark}{Remark}%
+\theoremstyle{thmstylethree}%
+\newtheorem{definition}{Definition}
```

Comment:
This is just boiler plate code from the standard OUP template that I personally would not use and so I wasn't sure how to best make it accessible to the user. It seems like this could go in `header-includes`, though it is quite a bit of code.

Reply:
`header-includes` from pandoc requires text but `pdf_document()` has a `includes` variable, with a `in_header` value that takes a file. So you could add such definition in a `preamble` tex file and pass it to the format

````
output: 
  oup_article:
    includes:
      in_header: preamble.tex
````

Reply:
I opted to drop it in the `header-includes`. It won't win an award for beauty, but it works. If it is an issue, I can move it out to a file as you suggest.